aws-provision.sh

```diff
@@ -61,4 +66,5 @@ sudo rm -rf /root/.ssh
 sudo rm -rf /home/packer/.ssh
 sudo rm -rf /home/ubuntu/.ssh
 
+podman system info
 itzo --version
```

Comment:
Is there a fluentd command that can validate a config file? It'd be nice to make sure that the config is good and fluentd will start up.

Reply:
Yes, added `--dry-run` for google and amazon builds :+1: